changelogs/fragments/1894-Gracefully_fail_when_remote_src_does_not_exist.yml

```
@@ -0,0 +1,3 @@
+trivial:
```

This is something we want users to know :) 
```suggestion
minor_changes:
```